website/content/partials/telemetry-metrics/vault/audit/log_response_failure.mdx

```diff
@@ -0,0 +1,14 @@
+### vault.audit.log_response_failure ((#vault-audit-log_response_failure))
+
+Metric type | Value   | Description
+----------- | ------- | -----------
+counter     | number  | Number of audit log request failures across all devices
+
+The number of request failures is a **crucial metric**.
+
+A non-zero value for `vault.audit.log_response_failure` indicates that one of
+the configured audit log devices failed to respond to Vault. Vault stops all
```

Comment:
Same update here as for log_request_failure please.

Reply:
done :)